### PR TITLE
[New Rule] Incoming DCOM Lateral Movement with MMC

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/487

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Comment by Samirbous** [comment]
Example of Alert:

image

File: rules/windows/lateral_movement_dcom_mmc20.toml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/06"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies use of Distributed Component Object Model (DCOM) to run commands from a remote host, which are launched via
+the MMC20 Application COM Object. This behavior may indicate an attacker abusing a DCOM application to stealthy move
+laterally.
```
Comment (bm11100) [inline rules/windows/lateral_movement_dcom_mmc20.toml:12]:
```suggestion
Identifies the use of Distributed Component Object Model (DCOM) to run commands from a remote host, which are launched via
the MMC20 Application COM Object. This behavior may indicate an attacker abusing a DCOM application to move
laterally.
```

**Review by bm11100 — APPROVED** [review]
lgtm

File: rules/windows/lateral_movement_dcom_mmc20.toml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+[metadata]
+creation_date = "2020/11/06"
+ecs_version = ["1.6.0"]
```
Comment (brokensound77) [inline rules/windows/lateral_movement_dcom_mmc20.toml:3]:
```suggestion
```

File: rules/windows/lateral_movement_dcom_mmc20.toml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+query = '''
+sequence by host.id with maxspan=1m
+ [network where event.type == "start" and process.name : "mmc.exe" and
+  source.port > 49151 and destination.port > 49151 and source.address != "127.0.0.1" and source.address != "::1" and
+  network.direction == "incoming" and network.transport == "tcp"
+ ] by process.entity_id
+ [process where event.type in ("start", "process_started") and
+  process.parent.name : "mmc.exe"
+ ] by process.parent.entity_id
+'''
```
Comment (brokensound77) [inline rules/windows/lateral_movement_dcom_mmc20.toml:35]:
```suggestion
query = '''
sequence by host.id with maxspan=1m
 [network where event.type == "start" and process.name : "mmc.exe" and
  source.port > 49151 and destination.port > 49151 and source.address not in ("127.0.0.1", "::1") and
  network.direction == "incoming" and network.transport == "tcp"
 ] by process.entity_id
 [process where event.type in ("start", "process_started") and process.parent.name : "mmc.exe"
 ] by process.parent.entity_id
'''
```

File: rules/windows/lateral_movement_dcom_mmc20.toml (hunk 2 above)
Comment (Samirbous) [reply rules/windows/lateral_movement_dcom_mmc20.toml:3]: bye bye ecs_version 😭

**Review by threat-punter — APPROVED** [review]
🎉

**Pull request merged**